aidatlu/constellation/README.md

```diff
@@ -0,0 +1,112 @@
+---
+# SPDX-FileCopyrightText: 2024 DESY and the Constellation authors
+# SPDX-License-Identifier: CC-BY-4.0 OR EUPL-1.2
+title: "AidaTLU"
+description: "Satellite for the AIDA-2020 TLU using a Python based control software"
+---
+
+## Description
+
+The AIDA-2020 Trigger Logic Unit is designed to proved flexible trigger configurations in test beam settings (https://doi.org/10.48550/arXiv.2005.00310).
+
+The Python-based control software for the AIDA-2020 TLU provides a comprehensive interface for controlling the TLU.
+The software establishes a connection to the hardware and allows easy configuration of different trigger setups.
+Information over each individual trigger signal is saved in a compressed and human-readable HDF5 format.
+
+The satellite to connect the AIDA-2020 TLU to the [Constellation](https://constellation.pages.desy.de/) control and data acquisition framework.
+
+
+## Building
+
+After installing [IPbus](https://ipbus.web.cern.ch/doc/user/html/software/install/compile.html), also install the Python bindings and possible constellation requirement from the top file in the [Aida-TLU](https://github.com/SiLab-Bonn/aidatlu) repository.
+
+```bash
+pip install .[constellation]
+```
+
+A more detailed description of the prerequisite can also be found [here](https://silab-bonn.github.io/aidatlu/Introduction.html#installation).
+
+## Usage
+
+Add the cactus library path:
+
+```bash
+export LD_LIBRARY_PATH=<install_location>/lib
+```
+
+You also need to start the control hub:
+```bash
+/ipbus-software/controlhub/scripts/controlhub_start
+```
+
+Finally, start the satellite with:
+```bash
+SatelliteAidaTLU
```

Comment:
Don't you need a `-g` or `-n` here? If so, add the necessary ones to the example as well